Libraries/LibWeb/HTML/HTMLTableColElement.idl

```diff
@@ -6,7 +6,7 @@ interface HTMLTableColElement : HTMLElement {
 
     [HTMLConstructor] constructor();
 
-    [CEReactions] attribute unsigned long span;
+    [CEReactions, Reflect, ReflectRange=(1, 1000), ReflectFallback=1] attribute unsigned long span;
```

Comment:
The existing implementation for this and the other IDL attributes changed in this commit should be removed, otherwise I don't think these changes will do anything.

Reply:
Unfortunately, these have implementations that intermingle with other parts of the system, so removing them is not easy.. Maybe this commit should be dropped for now until we can figure out a way to do that?